module/VuFindApi/src/VuFindApi/Controller/Factory.php

```diff
@@ -67,11 +66,10 @@ public static function getApiController(ServiceManager $sm)
      */
     public static function getSearchApiController(ServiceManager $sm)
     {
-        $recordFields = $sm->get('VuFind\Config\YamlReader')
-            ->get('SearchApiRecordFields.yaml');
-        $helperManager = $sm->get('ViewHelperManager');
-        $rf = new RecordFormatter($recordFields, $helperManager);
-        $controller = new SearchApiController($sm, $rf, new FacetFormatter());
-        return $controller;
+        return new SearchApiController(
```

Comment:
This refactoring of the RecordFormatter to a service seems like it might make sense as a separate commit, since it improves extensibility and has nothing to do with the rest of this PR. Is there any reason I shouldn't extract this part of the change and commit it to master?

Reply:
That'd be great, but then the same should probably be done with FacetFormatter.